utils/check-environment-variable.ts

```diff
@@ -10,3 +10,14 @@ export function checkEnvVariables(...args: string[]) {
         throw new Error(`The following environment variables are not set yet: ${missingVariables.join(', ')}. Please set them in .env file or pipeline environments.`);
     }
 };
+
+export function getEnv(variableName: string, defaultValue?: string) {
+    const variable = process.env[variableName];
+    if (!variable) {
+        if (defaultValue !== undefined) {
+            return defaultValue;
+        }
+        throw new Error(`${variableName} environment variable must be defined.`);
+    }
+    return variable
```

Comment:
**Suggestion:** Add a semicolon at the end of the `return variable` statement to maintain consistency and avoid potential issues.
> 🌴  **Medium Priority** 
 > [category: **`best practice`**, importance: **`5`**]
> **Reason:** Adding a semicolon enhances code consistency and prevents potential issues in JavaScript execution, although it's a minor syntax correction. 

```suggestion
    return variable;
```